sema/path_type.go

```diff
@@ -48,6 +51,9 @@ var StoragePathType = &SimpleType{
 	Comparable:    false,
 	Exportable:    true,
 	Importable:    true,
+	conformances: []*InterfaceType{
+		StructStringerType,
+	},
```

Comment:
The conformance list here for `StoragePath` and for the other subtypes of `Path` below, should not be necessary, as the subtypes should implicitly conform to the interface list as well.

I guess removing them wouldn't work though, because `SimpleType.EffectiveInterfaceConformanceSet` doesn't take subtyping relationships into account yet (and we do not have that available there, only implicitly in the `IsSubtype` function)